src/content/protocols/ethena.md

```diff
@@ -0,0 +1,158 @@
+---
+protocol: "Ethena"
+website: "https://ethena.fi/"
+x: "https://x.com/ethena_labs"
+github: "https://github.com/ethena-labs"
+defillama_slug: ["ethena"]
+chain: "Ethereum"
+stage: 0
+risks: ["L", "L", "H", "H", "M"]
+author: ["andrijdavid"]
+submission_date: "2024-12-10"
+publish_date: "1970-01-01"
+acknowledge_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+
+Ethena is a synthetic dollar protocol built on Ethereum that provides a crypto-native solution for money not reliant on traditional banking system infrastructure. It introduces the 'Internet Bond,' a globally accessible dollar-denominated rewards instrument. Ethena's synthetic dollar, USDe, is fully backed by staked crypto assets (Etherum and Bitcoin) and its corresponding short futures positions. Ethena employs delta-hedging strategies to maintain stability and generate yields for users.
+
+# Overview
+
+## Chain
+
+The Ethena protocol is deployed on Ethereum mainnet.
+
+> Chain score: L
+
+## Upgradeability
+
+Ethena's smart contracts are not upgradable but the ability to set and update roles introduces a level of flexibility and risk. 
+
+> Upgradeability: L
+
+## Autonomy
+
+Ethena's contracts have dependencies on external roles and addresses, such as the minter, redeemer, and custodian addresses. 
+
+> Autonomy score: H
+
+## Exit Window
+
+Ethena's contracts have processes for transferring admin rights and setting role without any exit window. 
+
+> Exit Window score: H
+
+## Accessibility
+
+Only a single user interface exists.
+
+> Accessibility score: M
+
+# Technical Analysis
+
+## Contracts
+
+| Contract Name | Address |
+| ------------- | ------- |
+| USDe    | [0x4c9EDD5852cd905f086C759E8383e09bff1E68B3](https://etherscan.io/address/0x4c9EDD5852cd905f086C759E8383e09bff1E68B3)   |
+| EthenaMinting    | [0xe3490297a08d6fC8Da46Edb7B6142E4F461b62D3](https://etherscan.io/address/0xe3490297a08d6fC8Da46Edb7B6142E4F461b62D3)   |
+| StakedUSDeV2  | [0x9D39A5DE30e57443BfF2A8307A4256c8797A3497](https://etherscan.io/address/0x9D39A5DE30e57443BfF2A8307A4256c8797A3497)    |
+| StackingRewardDistributor | [0xf2fa332bD83149c66b09B45670bCe64746C6b439](https://etherscan.io/address/0xf2fa332bd83149c66b09b45670bce64746c6b439) |
+| ENA | [0x57e114B691Db790C35207b2e685D4A43181e6061](https://etherscan.io/address/0x57e114B691Db790C35207b2e685D4A43181e6061) |
+| sENA | [0x8bE3460A480c80728a8C4D7a5D5303c85ba7B3b9](https://etherscan.io/token/0x8bE3460A480c80728a8C4D7a5D5303c85ba7B3b9) |
+
```

Comment:
please rename 

sENA (Proxy) | [0x8bE3460A480c80728a8C4D7a5D5303c85ba7B3b9] | ...

and introduce this row as well

sENA (Implementation | 0x7fd57b46ae1a7b14f6940508381877ee03e1018b | ...

Reply:
Solved